packages/@expo/cli/src/start/platforms/ios/AppleAppIdResolver.ts

```diff
@@ -4,6 +4,8 @@ import fs from 'fs';
 
 import { AppIdResolver } from '../AppIdResolver';
 
+const debug = require('debug')('expo:start:platforms:ios:AppleAppIdResolver') as typeof console.log;
```

Comment:
is the debug type missing here, or is this caused by the require?

Reply:
caused by require instead of using import